layouts/partials/top.html

```diff
@@ -0,0 +1,22 @@
+<div id="topbar" class="bg-dark text-white py-2">
+    <div class="container d-grid">
+        <!-- Left Side: Metrics -->
+        <div class="topbar-metrics">
+            <span class="topbar-metric me-4" title="Average peak traffic over the last 30 days">
+                📈 Peak Traffic (30d Avg): {{ .Site.Params.metrics.peak_traffic }}
```

Comment:
I would like to give more focus to these stats because it directly shows what we are about. An example from another IXP https://sthix.net/
I've been meaning do this. Having the the numbers count up from zero. Networks, Ports and Connected capacity we can do currently from the ixp-manager api (the one we use on the members page). Statistics of real throughput would have to come from prometheus.

![image](https://github.com/user-attachments/assets/7cb3ecb2-9009-4b7b-a101-452e45109736)

I think this could go into a new PR so we get out some of the other stuff faster.

Reply:
Totally agree, this is great! I was trying really hard on what could be relevant, but this is spot on from my perspective. I'll remove the metrics fully for now and let's push this for the next release.